docs/acp/user/intro-to-acp/acp-overview.md

```diff
@@ -26,19 +26,56 @@ The ACP has the following advantages:
 - Contains only approved and secure apps
 - Always up-to-date apps
 
+## Installing the ACP catalog
+
+ACP catalog is included by default to the Spryker Cloud product. However, if you started your Spryker project before March 31, 2022, to use the ACP catalog, you must install the following module into your Spryker project:
+
+* `spryker/app-catalog-gui: ^1.2.0` or higher
+* add the configuration for the module and its dependency
+
+<details open>
+<summary>config/Shared/config_default.php</summary>
+
+```php
+use Spryker\Shared\AppCatalogGui\AppCatalogGuiConstants;
+use Spryker\Shared\OauthAuth0\OauthAuth0Constants;
+use Spryker\Shared\Store\StoreConstants;
+use Spryker\Zed\OauthAuth0\OauthAuth0Config;
+
+$aopApplicationConfiguration = json_decode(html_entity_decode((string)getenv('SPRYKER_AOP_APPLICATION')), true);
+$aopAuthenticationConfiguration = json_decode(html_entity_decode((string)getenv('SPRYKER_AOP_AUTHENTICATION')), true);
+
+$config[AppCatalogGuiConstants::APP_CATALOG_SCRIPT_URL] = $aopApplicationConfiguration['APP_CATALOG_SCRIPT_URL'] ?? '';
+$config[AppCatalogGuiConstants::OAUTH_PROVIDER_NAME] = OauthAuth0Config::PROVIDER_NAME;
+$config[AppCatalogGuiConstants::OAUTH_GRANT_TYPE] = OauthAuth0Config::GRANT_TYPE_CLIENT_CREDENTIALS;
+$config[AppCatalogGuiConstants::OAUTH_OPTION_AUDIENCE] = 'aop-atrs';
+$config[OauthAuth0Constants::AUTH0_CUSTOM_DOMAIN] = $aopAuthenticationConfiguration['AUTH0_CUSTOM_DOMAIN'] ?? '';
+$config[OauthAuth0Constants::AUTH0_CLIENT_ID] = $aopAuthenticationConfiguration['AUTH0_CLIENT_ID'] ?? '';
+$config[OauthAuth0Constants::AUTH0_CLIENT_SECRET] = $aopAuthenticationConfiguration['AUTH0_CLIENT_SECRET'] ?? '';
+$config[StoreConstants::STORE_NAME_REFERENCE_MAP] = $aopApplicationConfiguration['STORE_NAME_REFERENCE_MAP'] ?? [];
+```
+</details>
+
+
+{% info_block warningBox "" %}
+
+Your project must be hosted in Spryker Cloud.
+
+{% endinfo_block %}
+
 ## Accessing the ACP catalog
 
-The ACP catalog is a page inside the Back Office that contains the list of applications you can connect to your shop.
-You can access the ACP catalog only if you are the SCCOS customer. If you were onboarded after March 31st, 2022, you get the ACP catalog integrated into your Back Office by default. If you were onboarded earlier, [contact us](https://support.spryker.com/). Once you complete the installation, the ACP catalog appears in the Back Office:
+The ACP catalog is a page inside the Back Office containing the application list you can connect to your shop.
+You can access the ACP catalog only if you are a SCCOS customer. If you were onboarded after March 31, 2022, you get the ACP catalog into your Back Office by default. If you were onboarded earlier, [contact us](https://support.spryker.com/). Once you complete the installation, the ACP catalog appears in the Back Office:
 
 ![aop-catalog](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/app-orchestration-platform-overview/aop-catalog.png)
```

Comment:
fresh version of the Catalog look 
![image](https://user-images.githubusercontent.com/454530/206717329-d5a7636b-451d-44b6-9901-5f80831ce627.png)